src/js/angular-tooltips.js

```diff
@@ -77,12 +79,7 @@
 
         thisElement.bind(showTriggers, function onMouseEnterAndMouseOver() {
 
-          if (!initialized) {
-
-            initialized = true;
-            $scope.initTooltip(side);
-          }
-
+          $scope.initTooltip(side);
```

Comment:
This is a good point i didn't thought about before, imho the easy way is to check somehow when the element moves on the screen probably, thank you :)